src/maker/api.rs

```diff
@@ -522,12 +528,26 @@ impl Maker {
 /// If any one of them is ever observed, run the recovery routine.
 pub(crate) fn check_for_broadcasted_contracts(maker: Arc<Maker>) -> Result<(), MakerError> {
     let mut failed_swap_ip = Vec::new();
+    let tracker_addr = {
+        let guard = maker.tracker.read()?.clone();
+        guard.clone()
+    };
+    let mut tracker_stream = match TcpStream::connect(tracker_addr.unwrap()) {
```

Comment:
We need to feature gate the connection for clearnet for tests, and tor for production.